app/global-translations/locale-en.json

```diff
@@ -1,5 +1,33 @@
 {
   "#Common": "....",
+  "label.heading.startdate": "Start Date",
+  "label.heading.description": "Description",
+  "label.heading.fromdate": "From Date",
+  "label.heading.interestrate": "Interest Rate",
+  "label.input.name": "Name",
+  "label.anchor.importclients":"Clients",
+  "label.anchor.importcenters":"Centers",
+  "label.selectentity": "Select entity",
+  "label.selectaction": "Select action",
+  "label.heading.address": "Address",
+  "label.input.status": "Status",
+  "label.heading.editsmscampaign": "Edit SMS Campaign",
+  "label.heading.meetingdetails": "Meeting Details",
+  "label.heading.createdby": "Created By",
+  "label.button.addAddress": "Add Address",
+  "label.heading.campaign": "Campaign",
+  "label.input.city": "City",
+  "label.tooltip.incomefromfees": "An Income account which is credited when a fee is paid by account holder on this account",
+  "label.tooltip.incomefrompenalties": "An Income account, which is credited when a penalty is paid by account holder on this account",
+  "label.tooltip.daysinyear": "The setting for number of days in year to use to calculate interest",
+  "label.tooltip.marketprice": "Price of each share",
+  "label.anchor.editpermissions": "Edit Permissions",
+  "label.input.email": "Email",
+  "label.anchor.importoffices":"Offices",
+  "label.heading.isactive": "Status",
+  "label.input.alias": "Alias",
+  "label.input.incomefrominterest": "Income from Interest",
+  "": "Index",
```

Comment:
Under #Common section, you see that we have #Headings sub section with all heading labels, #Anchors sub section with anchor labels, #Inputs sub section for input labels and so on. Follow this format and ensure the respective labels are under the correct sub section of a section respectively. In case you see some other label under incorrect sub section move that under the correct sub section as well.